Sell/Product/ProductGettable.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  ProductGettable.swift
```

Comment:
I'm not at all sure about this name, any other suggestions?

Reply:
Fetchable? retrievable?